src/GDriveCloudStorage.js

```diff
@@ -0,0 +1,34 @@
+import GDrive from 'react-native-google-drive-api-wrapper';
+
+export async function setItem(keyId, value) {
+  const folderId = await GDrive.files.safeCreateFolder({ name: 'Photon', parents: ['root'] });
+
+  return GDrive.files.createFileMultipart(
+    '',
+    'text/plain',
+    {
+      parents: ['root', folderId],
+      name: keyId,
+      appProperties: {
+        keyId: value,
+      },
+    },
+    false,
+  );
+}
+
+export async function getItem(keyId) {
+  try {
+    const fileId = await GDrive.files.getId(keyId, ['Photon'], 'text/plain', false);
+    const meta = await GDrive.files.get(fileId);
+
```

Comment:
Remove new line here

Reply:
We should add in a formatter like prettier to avoid these bubbling up to PR's I think

Reply:
There should already be an `.eslintrc` which extends prettier. linting runs on travis. But it seems like there is still too much wiggle room in this case agreed.